### PR TITLE
fix default value of pipelined lock

### PR DESCRIPTION
**Pull request by you06**
Signed-off-by: you06 <you1474600@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

The default value of pipelined lock is wrong.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by you06** [comment]
@sticnarf PTAL here also.

**Comment by ran-huang** [comment]
This PR is translated from https://github.com/pingcap/docs-cn/pull/5985

/lgtm
/translation from-docs-cn
/hold
/label require-LGT1

**Comment by sticnarf** [comment]
LGTM

**Comment by you06** [comment]
> 
> 
> /label v5.0

I think this does not need to patched to 5.0

File: faq/deploy-and-maintain-faq.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ Check the time difference between the machine time of the monitor and the time w
 >
 > It is not recommended to deploy TiDB using TiDB Ansible since TiDB v4.0. [Use TiUP to deploy TiDB](/production-deployment-using-tiup.md) instead.
 
-If the central control machine cannot access the Internet, you can [deploy TiDB offline using TiDB Ansible](https://docs.pingcap.com/tidb/stable/offline-deployment-using-ansible).
+If the central control machine cannot access the Internet, you can [deploy TiDB offline using TiDB Ansible](https://docs.pingcap.com/tidb/v4.0/offline-deployment-using-ansible).
```
Comment (TomShawn) [inline faq/deploy-and-maintain-faq.md:113]:
```suggestion
If the central control machine cannot access the Internet, you can [deploy TiDB offline using TiDB Ansible](/offline-deployment-using-ansible.md).
```

Comment (you06) [reply faq/deploy-and-maintain-faq.md]: It's solved by rebase

**Pull request merged**